openapi-contracts/src/main/com/sinch/sdk/domains/conversation/models/v1/credentials/InstagramCredentialsImpl.java

```diff
@@ -56,7 +56,7 @@ public OptionalValue<String> businessAccountId() {
     return businessAccountId;
   }
 
-  /** Return true if this InstagramCredentials object is equal to o. */
+  /** Return true if this Instagram_Credentials object is equal to o. */
```

Comment:
Name/comment is coming from generator tool "out of the box"
Need to enhance the generator 
See PR's comment:

> Synch with latest
> ...
>     latest code generator tool
> This PR is sync point for feature branch
> It just provide an intermediate state because of:
> ...